Automattic/wp-calypso
# Licensing Portal: Add an Invoices section

PT: pdpAdu-bj-p2
Card: 1201902532330136-as-1201991679661439
Requires D77100-code.

#### Changes proposed in this Pull Request

* Add a new "Invoices" section to the Licensing Portal.

![Screen Shot 2022-04-08 at 18 40 38](https://user-images.githubusercontent.com/22746396/162476088-5b31ba91-680c-48f4-875a-d42fbcf07272.png)

#### Testing instructions

* Apply patch and run Jetpack Cloud
* Open http://jetpack.cloud.localhost:3000/partner-portal/invoices
* Confirm you're seeing your test partner key's invoices and that the UI works as expected.
* Try testing with a different partner key by adjusting the D77100-code source code to pull invoices that have various totals and statuses.
